challenge.md

```diff
@@ -6,19 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+git clone + SSH URL
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+git log
+
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
 
-4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What you we do? What git command would one use?
+git checkout + file name
+
+4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What do we do? What git command would one use?
 
+git checkout -b to create a new branch
 
-5. I want to add a file to the staging area. What git commit would I use?
+5. I want to add a file to the staging area. What git command would I use?
 
+git add + file
```

Comment:
To be precise, `git add {path-to-file}`. As it happens, though, a file's name can also double as a relative path to that file, provided that it's in the root of the project.